sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -32,6 +32,31 @@ export class InstitutionLocationFormAPIInDTO extends AddressDetailsAPIInDTO {
   primaryContactPhone: string;
 }
 
+export class InstitutionLocationPrimaryContactAPIInDTO {
+  @Allow()
+  primaryContactFirstName: string;
+  @Allow()
+  primaryContactLastName: string;
+  @Allow()
+  primaryContactEmail: string;
+  @Allow()
+  primaryContactPhone: string;
+}
+export class AESTInstitutionLocationAPIInDTO extends AddressDetailsAPIInDTO {
```

Comment:
since its the same form you can extend `InstitutionLocationPrimaryContactAPIInDTO`

Reply:
While using allow, I would suggest to use a comment to mention that we aren't using strict class validators here due to dry-run submission check.

Reply:
I saw the service method and we are not running dry-run validation. 
If dry-run is not required/not used then we can use @IsNotEmpty() for mandatory fields.